otherlibs/threads/pervasives.ml

```diff
@@ -177,6 +177,8 @@ let min_float =
 let epsilon_float =
   float_of_bits 0x3C_B0_00_00_00_00_00_00L
 
+let pi = 0x3.243f6a8885a308d313198a2e037073
```

Comment:
Is there more precision in this definition than what OCaml doubles can represent?

```ocaml
# Printf.sprintf "%h" 0x3.243f6a8885a308d313198a2e037073;;
- : string = "0x1.921fb54442d18p+1"
```

Do `0x3.243f6a8885a308d313198a2e037073` and `0x1.921fb54442d18p+1` represent different floating-point number? The latter is also exactly what I get on my machine with `4. *. atan 1.` (but maybe `atan`'s precision depends on the system and shouldn't be relied on for a definition?).